### ISSUE TITLE
The framework, in its current form, is completely unusable.

### ISSUE THREAD

**Issue by jmpr0** [OrangeUW/AutoML4ETC#2]
This repository contains code with functionality that is difficult to understand, particularly the loading of the MNIST dataset and the use of its labels as targets for the NAS procedure.

**Comment by navidpadid** [comment]
hk.search uses these inputs: (train_dataset, y_train1, test_dataset, y_test1) the y_train1 and y_test1 is from MNIST, but the train_dataset and test_dataset have both the INPUT and LABELS. The Keras implementation supports both giving INPUT and LABELS separately or together. Here, the MNIST is just a place holder (eg. dummy values) since the train_dataset and test_dataset has both the INPUT and LABELS; just used the MNIST out of the keeping the code compatible. The MNIST labes are DISCARDED. To understand the NAS procedure I suggest you take a look at the hk.search method mentioned.

**Comment by navidpadid** [comment]
closing the issue based on the above explanation.
